tests/unit/browser/test_adblock.py

```diff
@@ -88,15 +88,21 @@ class FakeDownloadManager:
 
     def __init__(self, tmpdir):
         self._tmpdir = tmpdir
+        self.downloads = []
 
     def get(self, url, target, **kwargs):
         """Return a FakeDownloadItem instance with a fileobj.
 
         The content is copied from the file the given url links to.
         """
+        if isinstance(target, downloads.FileDownloadTarget):
```

Comment:
I think this could be a context manager? Something like:

```python
@contextlib.contextmanager
def _open_fileobj(self, target):
    if isinstance(target, downloads.FileDownloadTarget):
        target.fileobj = open(target.filename, 'wb')
        try:
            yield
        finally:
            target.fileobj.close()
    else:
        yield
```

That'd make `get` a bit cleaner, and also clean things up correctly if tests failed.